### DISCUSSION TITLE
Seed invalid integer

### DISCUSSION THREAD

**Discussion by whitneyt1** [angelolab/ark-analysis#851] [Q&A]
Hi! 

While pixel clustering, I encountered the following error: Error in set.seed(): seed invalid integer
<img width="1335" alt="Screen Shot 2022-12-06 at 1 53 26 PM" src="https://user-images.githubusercontent.com/46573884/206032787-b327de90-8fd4-4d29-b07f-b735583094ae.png">

When I tried manually setting seed value to 42, i received the same error. 

I corrected this by changing the pixel_consesus_cluster.R file to: 
<img width="239" alt="Screen Shot 2022-12-06 at 2 03 06 PM" src="https://user-images.githubusercontent.com/46573884/206032955-24c1bfa0-993c-434d-9a12-eaf64f80e74e.png">

instead of it calling the arg[11] from the pixel_cluster_utils.py file, which reads in str(seed). 
<img width="720" alt="Screen Shot 2022-12-06 at 2 04 28 PM" src="https://user-images.githubusercontent.com/46573884/206033204-4df54d68-ee57-4d6b-bf2c-6764b28fc9c5.png">

This might be being address in #839, but just FYI! 

thanks!

**Comment by ngreenwald** [comment]
Yes, we'll have this resolved once #839 is merged in, which should be this week. Thanks for tracking down the problem. 

https://github.com/angelolab/ark-analysis/issues/852
